### PR TITLE
scmi: Prepare v0.2.0 release

### PR DESCRIPTION
**Pull request by slp**
Update CHANGELOG.md and Cargo.toml for v0.2.0 relase.

### CODE REVIEW

**Comment by slp** [comment]
CI failure doesn't seem to be related to this PR:

```

test audio_backends::pipewire::tests::test_pipewire_backend_invalid_stream ... FAILED
--
  |  
  | failures:
  |  
  | ---- audio_backends::pipewire::tests::test_pipewire_backend_invalid_stream stdout ----
  | INFO: dbus_session_bus_address=unix:path=/tmp/.tmpG1gbPk/dbus
  | INFO: Wait for dbus to setup...
  | INFO: Launch pipewire.
  | INFO: Wait for pipewire to setup...
  | thread 'audio_backends::pipewire::tests::test_pipewire_backend_invalid_stream' panicked at vhost-device-sound/src/audio_backends/pipewire.rs:97:42:
  | Failed to connect to core: CreationFailed
  | INFO: Killing pipewire pid 4538
  | ERROR: pipewire stderr [E][07173.540058] mod.rt       \| [     module-rt.c:  236 pw_rtkit_bus_get()] Failed to connect to system bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
  | [W][07173.540068] mod.rt       \| [     module-rt.c: 1055 pipewire__module_init()] Realtime scheduling disabled: unsufficient realtime privileges, Portal not found on session bus, and no system bus for RTKit: Connection refused
  | [W][07173.543531] default      \| [        thread.c:  101 impl_acquire_rt()] acquire_rt thread:0x759284bcb640 prio:-1 not implemented
```

Seems like that pipewire test depends on a running system-wide dbus. Is that expected to work in a buildkite container?

**Comment by epilys** [comment]
@slp IIRC this is a transient failure, it only fails sometimes. We'd need to re-run the CI.

**Review by epilys — APPROVED** [review]

**Comment by stefano-garzarella** [comment]
> @slp IIRC this is a transient failure, it only fails sometimes. We'd need to re-run the CI.

yep, just restarted and everything is fine.

**Review by stefano-garzarella — APPROVED** [review]

**Comment by stefano-garzarella** [comment]
I'll tag it and release in a couple of minutes

File: vhost-device-scmi/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,15 @@
 
 ### Deprecated
 
+## [0.2.0]
+
+## Changed
```
Comment (stefano-garzarella) [inline vhost-device-scmi/CHANGELOG.md:14]: Here it should be `### Changed`, but I already released it, we can fix next time ;-)
Just noticed while creating the release on github.

Comment (slp) [reply vhost-device-scmi/CHANGELOG.md:14]: Oops...

Comment (stefano-garzarella) [reply vhost-device-scmi/CHANGELOG.md:14]: fixed here: https://github.com/rust-vmm/vhost-device/pull/695 ;-)

**Pull request merged**